rpc/legacyrpc/methods.go

```diff
@@ -3104,6 +3107,128 @@ func signRawTransactions(s *Server, icmd interface{}) (interface{}, error) {
 	return &dcrjson.SignRawTransactionsResult{Results: toReturn}, nil
 }
 
+// startAutoBuyer handles the startautobuyer command.
+func startAutoBuyer(s *Server, icmd interface{}) (interface{}, error) {
+	cmd := icmd.(*dcrjson.StartAutoBuyerCmd)
+	w, ok := s.walletLoader.LoadedWallet()
+	if !ok {
+		return nil, &ErrUnloadedWallet
+	}
+
+	config := s.ticketbuyerConfig
+
+	if cmd.BalanceToMaintain != nil {
+		if *cmd.BalanceToMaintain < 0 {
+			e := fmt.Errorf("balancetomaintain (%d) must be non-negative", *cmd.BalanceToMaintain)
```

Comment:
this is a float and can't be formatted with %d.  it would be simpler to use %v everywhere, or even convert them into amounts and call their string method